Parsers/Find Paper.js

```diff
@@ -0,0 +1,42 @@
+/*
+activation_example:!findpaper paper details
+regex:!findpaper
+flags:
+*/
+
+// Extracting the search query
+var searchQuery = current.text.replace(/!findpaper/gmi, "").trim().substring(0, 1000);
+
+// Searching for a research paper
+var researchPaperRequest = new sn_ws.RESTMessageV2();
+researchPaperRequest.setEndpoint('https://api.semanticscholar.org/graph/v1/paper/search?query=' + encodeURIComponent(searchQuery) + '&limit=1');
+researchPaperRequest.setHttpMethod("GET");
+researchPaperRequest.setRequestHeader("Accept", "application/json");
+
+// Executing the API call
+try {
+    var response = researchPaperRequest.execute();
+    var responseBody = response.getBody();
+    var responseJson = JSON.parse(responseBody);
+    
+    // Extracting details of the first paper found
+    if (responseJson && responseJson.data && responseJson.data.length > 0) {
+        var paper = responseJson.data[0];
+        var title = paper.title || "Title not available";
+        var authors = paper.authors.map(author => author.name).join(", ") || "Authors not available";
```

Comment:
As commented, validating arrow function support

Reply:
Thanks for reviewing

Reply:
I have confirmed that this will not run in the SlackerBot scope in its current state as arrow function notation is not supported in the ES5 mode on this instance. Please update this to a function declaration as per the example I have mentioned